cluster-autoscaler/simulator/cluster_test.go

```diff
@@ -51,14 +51,19 @@ func TestFindPlaceAllOk(t *testing.T) {
 		"n1": true,
 		"n2": true,
 	}
+	deleteOptions := NodeDeleteOptions{
```

Comment:
nit: since all test cases here use identical `NodeDeleteOptions`, it might make sense to just wrap them into some `testDeleteOptions()` func instead of defining separately everywhere.